package.json

```diff
@@ -155,6 +155,7 @@
     "eslint-plugin-simple-import-sort": "^12.1.1",
     "eslint-plugin-sonarjs": "^1.0.3",
     "eslint-plugin-unicorn": "^54.0.0",
+    "fzstd": "^0.1.1",
```

Comment:
I am emitting some security doubt
- the fzstd github repository[^1] has 21 commits, 
- 5 total contributors with 1-3 commits each
- is using `@types/node^ 14.11.2`[^2]
- 3 releases

[^1]: https://github.com/101arrowz/fzstd
[^2]: https://github.com/101arrowz/fzstd/network/dependencies

Reply:
@axel7083 I didn't find any better alternative not relying on the system dependency (requiring to install some native packages)

Reply:
https://facebook.github.io/zstd/#other-languages

Reply:
I understand the issue, but I still have some concern; we do not really have security guidelines to follow, so I will not hold it.

Reply:
note that the author of the lib is the author of a library that is used a lot
example https://www.npmjs.com/package/fflate (4M / week of download)

Reply:
This is not toward the author specifically, but linked to how small repository could be easy compromised and end up in a lot of machine if we ship it #XZUtils